fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/postgresql/SetPostgresOptimizerOptions.java

```diff
@@ -22,11 +22,11 @@
 
     private static final Logger log = Logger.getLogger(SetPostgresOptimizerOptions.class.getName());
 
-    // Configuration constants
+    // Configuration constants. Using 7 as the default value based on testing.
     private static final String JOIN_COLLAPSE_LIMIT = "join_collapse_limit";
-    private static final int DEFAULT_JOIN_COLLAPSE_LIMIT = 16;
+    private static final int DEFAULT_JOIN_COLLAPSE_LIMIT = 7;
     private static final String FROM_COLLAPSE_LIMIT = "from_collapse_limit";
-    private static final int DEFAULT_FROM_COLLAPSE_LIMIT = 16;
+    private static final int DEFAULT_FROM_COLLAPSE_LIMIT = 7;
```

Comment:
marked PR as in Review

Reply:
is there any corresponding Markdown update?

Reply:
Yes, doing that now.

Reply:
Actually, I may get Lee to change it because he's done some reorg of the user's guide and that sounds like a merge problem.

Reply:
updated the users' guide